### ISSUE TITLE
The staging app is crashing on Upgrade.

### ISSUE THREAD

**Issue by cakesoft-swati** [bithyve/hexa#3014]
-My old version was 1.5.0 (248) I went to testflight and upgraded it to 1.5.0 (250)., After that I entered the PIN and pressed the "Proceed" button, the app crashed.

**Comment by AliMeer** [comment]
@cakesoft-swati does upgrade crash from 1.4.6 to 1.5.0(250) ?

**Comment by cakesoft-swati** [comment]
Yes, it crashes even when upgrading from 1.4.6 to 1.5.0.

**Comment by Parsh** [comment]
I've tried upgrading from 1.4.6 to 1.5.0(on the development branch) and yeah the app crashed at multiple places.

The initial crash was on the home screen as the level health object wasn't available on the sss object while the home screen was trying to pull out the healthCheckInitializeKepper flag from it.

![Simulator Screen Shot - iPhone 12 mini - 2021-03-26 at 15 29 30](https://user-images.githubusercontent.com/17980466/112617730-5a15c680-8e4b-11eb-97a9-667ead069788.png)

After fixing that I was able to land on the home screen however the react-native debugger was bombarded with errors related to newBHR(as far as I understand). 

<img width="1440" alt="Screenshot 2021-03-26 at 3 30 06 PM" src="https://user-images.githubusercontent.com/17980466/112617994-ad881480-8e4b-11eb-8930-505c75a0585f.png">

<img width="1440" alt="Screenshot 2021-03-26 at 3 30 10 PM" src="https://user-images.githubusercontent.com/17980466/112618016-b4af2280-8e4b-11eb-8933-4baa0cf90d66.png">


Then the app was also crashing on the manage backup screen

<img width="1440" alt="Screenshot 2021-03-26 at 3 30 19 PM" src="https://user-images.githubusercontent.com/17980466/112617856-85001a80-8e4b-11eb-9ed2-9d57626f489f.png">


Conclusion: We might need to write an appropriate upgrade script in order to handle these newBHR related upgrade crashes.
